manifest-el9-shared.yaml

```
@@ -8,6 +8,9 @@ remove-from-packages:
   - - zram-generator
     - "/usr/lib/systemd/zram-generator.conf"
 
+ostree-layers:
+  - overlay/25rhcos-azure-udev
```

Having the same comment from below here would be nice:

```suggestion
  # The azure-vm-utils package provides these udev rules[1], but it wont be added
  # until EL10 [1]. This can be dropped when moving to EL10, provided the
  # package is included at that time.
  - overlay/25rhcos-azure-udev
```